docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/amazon-aws.md

```diff
@@ -196,37 +196,37 @@ chmod 400 ~/.ssh/ec2-strapi-key-pair.pem
 #### 2. Log in to your server as the default `ubuntu` user:
 
 ::: tip
-In the future, each time you log into your `EC2` server, you will need to add the path to the .pem file, e.g. `ssh -i ~/.ssh/ec2-strapi-key-pair.pem ubuntu@12.123.123.11`.
+In the future, each time you log into your `EC2` server, you will need to add the path to the .pem file and add the IP address for your EC2 instance at the end, e.g. `ssh -i ~/.ssh/ec2-strapi-key-pair.pem ubuntu@34.182.83.134`.
 :::
 
 ```bash
-ssh -i ~/.ssh/ec2-strapi-key-pair.pem ubuntu@12.123.123.11
+ssh -i ~/.ssh/ec2-strapi-key-pair.pem ubuntu@34.182.83.134
 
-Welcome to Ubuntu 18.04.2 LTS (GNU/Linux 4.15.0-1032-aws x86_64)
+Welcome to Ubuntu 22.04.2 LTS (GNU/Linux 4.15.0-1032-aws x86_64)
 
 ...
 
 To run a command as administrator (user "root"), use "sudo <command>".
 See "man sudo_root" for details.
 
-ubuntu@ip-12.123.123.11:~$
+ubuntu@ip-34.182.83.134:~$
 
 ```
 
 #### 3. Install **Node.js** with **npm**:
 
-Strapi currently supports `Node.js v12.x.x`. The following steps will install Node.js onto your EC2 server.
+Strapi currently supports `Node.js v16.x.x`. The following steps will install Node.js onto your EC2 server.
```

Comment:
We aren't recommending node 16 without installing yarn so this should probably be changed to node 14